### ISSUE TITLE
Impaired renal tubular reabsorption of uric acid

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6867]
**Preferred term label:**
Impaired renal tubular reabsorption of uric acid

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of uric acid.
PMID: 31474092

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated term**
Renal Fanconi syndrome HP:0001994

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
@RichardKlaus 
adding this term. I added this comment 

Uric acid excretion rate is the product of the filtered load (which can be approximated as plasma ultrafilterable urate multiplied by tge glomerular filtration rate) and fractional excretion of urate (FEUA), which represents the percentage of filtered urate that is excreted in the final urine. 

for additional information. Please reopen if this is not good.
